lib/client.js

```diff
@@ -43,9 +43,9 @@ $(document).ready(function () {
     var msg = JSON.parse(message);
     var html = '<li class=\'row\'>';
 
+    html += '<span class=\'name\'>' + msg.n + '</span>';
     html += '<small class=\'time\'>' + getTime(msg.t) + ' </small>';
-    html += '<span class=\'name\'>' + msg.n + ': </span>';
-    html += '<span class=\'msg\'>' + msg.m + '</span>';
+    html += '<p class=\'msg\'>' + msg.m + '</p>';
```

Comment:
I actually _like_ having the date/time first on the line (_before_) the name because it's a fixed width whereas the name won't be ... so the UI ends up looking more _aligned_ ... @yurm04 what were your _thoughts_ on this? 
is this the UI _other_ chat apps are implementing? 
(_genuine question ... I'm in a "Gitter" bubble so I don't know ..._)